content/etc/icingaweb2/config.ini

```diff
@@ -0,0 +1,10 @@
+[global]
+show_stacktraces = "1"
+config_backend = "db"
+config_resource = "icingaweb_db"
+module_path = "/usr/share/icingaweb2/modules:/etc/icingaweb2/modules"
```

Comment:
I use the findmypast's module, but you also need to use the official module to send the data to carbon